src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: